scripts/asm-installer/tests/run_basic_suite_managed

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+set -CeEu
+set -o pipefail
+
+SPATH="$(readlink -f "$0")"
+SDIR="$(dirname "${SPATH}")"; export SDIR;
+
+# shellcheck source=common.sh
+. "${SDIR}/common.sh"
+
+cd "${SDIR}"
+
+main() {
+  # CLI setup
+  parse_args "$@"
+
+  # Cluster setup
+  if [[ -n "${SERVICE_ACCOUNT}" ]]; then
+    echo "Authorizing service acount..."
+    auth_service_account
+  fi
+
+  local CLUSTER_NAME; CLUSTER_NAME="basic-suite-managed-${BUILD_ID::8}";
+  local NAMESPACE; NAMESPACE="namespace-${BUILD_ID}"
+  echo "Creating cluster ${CLUSTER_NAME}..."
+  create_working_cluster "${PROJECT_ID}" "${CLUSTER_NAME}" "${CLUSTER_LOCATION}"
+  # this trap isn't tested for all circumstances so caveat emptor
+  trap 'cleanup "${PROJECT_ID}" "${CLUSTER_NAME}" "${CLUSTER_LOCATION}" "${NAMESPACE}"; exit 1;' ERR
+
+  # Demo app setup
+  echo "Installing and verifying demo app..."
+  install_demo_app "${NAMESPACE}"
+
+  local GATEWAY; GATEWAY="$(kube_ingress "${NAMESPACE}")";
+  verify_demo_app "$GATEWAY"
+
+  # Test starts here
+  echo "Installing ASM with MeshCA..."
+  if [[ -n "${SERVICE_ACCOUNT}" ]]; then
+    echo "../install_asm \
+      -l ${CLUSTER_LOCATION} \
+      -n ${CLUSTER_NAME} \
+      -p ${PROJECT_ID} \
+      -m install \
+      --managed \
+      -c mesh_ca \
+      -s ${SERVICE_ACCOUNT} \
+      -k ${KEY_FILE} -v -e"
+    ../install_asm \
+      -l "${CLUSTER_LOCATION}" \
+      -n "${CLUSTER_NAME}" \
+      -p "${PROJECT_ID}" \
+      -m install \
+      --managed \
+      -c mesh_ca \
+      -s "${SERVICE_ACCOUNT}" \
+      -k "${KEY_FILE}" -v -e
+  else
+    echo "../install_asm \
+      -l ${CLUSTER_LOCATION} \
+      -n ${CLUSTER_NAME} \
+      -p ${PROJECT_ID} \
+      -m install \
+      --managed \
+      -c mesh_ca -v -e"
+    ../install_asm \
+      -l "${CLUSTER_LOCATION}" \
+      -n "${CLUSTER_NAME}" \
+      -p "${PROJECT_ID}" \
+      -m install \
+      --managed \
+      -c mesh_ca -v -e
+  fi
+
+  sleep 5
+  echo "Installing Istio manifests for demo app..."
+  install_demo_app_istio_manifests "${NAMESPACE}"
```

Comment:
Why do we need to re-install the app in the same namespace? Simply re-label the namespace is not enough?

Reply:
This doesn't reinstall the app, just the Istio objects to configure ingress etc.